subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/algorithm/hived_algorithm.go

```diff
@@ -734,14 +985,34 @@ func (h *HivedAlgorithm) lazyPreemptAffinityGroup(
 		Preemptor:      preemptor,
 		PreemptionTime: meta.Now(),
 	}
+
+	for _, podPhysicalPlacements := range victim.physicalGpuPlacement {
+		for _, podPhysicalPlacement := range podPhysicalPlacements {
+			for _, gpu := range podPhysicalPlacement {
+				if gpu != nil {
+				}
+			}
+		}
+	}
```

Comment:
Where is "usage" used? remove the whole?

Reply:
sorry... I saw it wrong